PennMobile/Dining/Model/Venues/DiningVenueName.swift

```diff
@@ -10,29 +10,33 @@ import Foundation
 
 enum DiningVenueName: String {
     case commons =       "1920 Commons"
-    case mcclelland =    "McClelland"
-    case nch =           "NCH Dining"
+    case mcclelland =    "McClelland Express"
+    case nch =           "New College House"
     case hill =          "Hill House"
     case english =       "English House"
-    case falk =          "Falk Kosher"
-    case frontera =      "Tortas Frontera"
-    case gourmetGrocer = "Gourmet Grocer"
+    case falk =          "Falk Kosher Dining"
+    case frontera =      "Tortas Frontera at the ARCH"
```

Comment:
What is the effect of this extra long name on the regular dining cell on the Dining Page? DiningCell.swift only uses the shortened name on the homepage.

Reply:
if I don't use the full name, the dictionary in storage (DiningVenue.swift init ) will not match it up, and the id won't get pushed to the server.

Reply:
I understand. I guess the question is do we need to use the short names in the Dining Page as well as in the Home Page because some are so much longer now. Alternatively, we could create a "Normal Names" enum/function which would handle special cases like Frontera where the name on the server is too long even for the full page. 

I think we're ok not worrying about this for now, but it's definitely something we should fix down the road, not to mention overhauling the venue identification system. It shouldn't break every time they change the names.